docs/concepts/alerts.md

```diff
@@ -0,0 +1,138 @@
+
+These values can be modified by the [support team](mailto:support@iguazio.com).
+
+## SDK
+
+The SDK supports these alert operations:
+
+- {py:func}`~mlrun.projects.MlrunProject.store_alert_config` &mdash; Create/modify an alert.
+- {py:func}`~mlrun.projects.MlrunProject.get_alert_config` &mdash;  Retrieve an alert.
+- {py:func}`~mlrun.projects.MlrunProject.reset_alert_config` &mdash; Reset an alert.
+- {py:func}`~mlrun.projects.MlrunProject.delete_alert_config` &mdash; Delete an alert.
+- {py:func}`~mlrun.projects.MlrunProject.get_alert_template` &mdash; Retrieve a specific alert template.
+- {py:func}`~mlrun.projects.MlrunProject.list_alert_templates` &mdash; Retrieve the list of all alert templates.
+- {py:func}`~mlrun.projects.MlrunProject.list_alerts_configs` &mdash; Retrieve the list of alerts of a project.
+
+## Alert templates
+Alert templates simplify the creation of alerts by providing a predefined set of configurations. The system comes with several 
+predefined templates that can be used with MLRun applications. 
+If you use non-MLRun applications (for example, with model monitoring), you must configure an application-specific alert. 
+The templates are cross-project objects. When generating an alert, you must assign the project to it. 
+See the {py:meth}`alert template parameters<mlrun.common.schemas.alert.AlertTemplate>`.
+
+## Predefined alerts
+The predefined alert types are:
+- DATA_DRIFT_DETECTED &mdash; A detected change in model input data that potentially leads to model performance degradation. See {ref}`monitoring-overview`.
+- DATA_DRIFT_SUSPECTED &mdash; A suspected change in model input data that potentially leads to model performance degradation. See {ref}`monitoring-overview`.
+- CONCEPT_DRIFT_DETECTED &mdash; A detected change, over time, of  statistical properties of the target variable (what the model is predicting). See {ref}`monitoring-overview`.
+- CONCEPT_DRIFT_SUSPECTED &mdash; A suspected change, over time, of  statistical properties of the target variable (what the model is predicting). See {ref}`monitoring-overview`.
+- MODEL_PERFORMANCE_DETECTED &mdash; A detected change of the overall model performance and/or feature-level performance. See {ref}`monitoring-overview`.
+- MODEL_PERFORMANCE_SUSPECTED &mdash; A suspected change of the overall model performance and/or feature-level performance. See {ref}`monitoring-overview`.
+- MODEL_SERVING_PERFORMANCE_DETECTED &mdash; A detected change in how much time the prediction takes (i.e. the latency, measured in time units).
+- MODEL_SERVING_PERFORMANCE_SUSPECTED &mdash; A suspected change in how much time the prediction takes (i.e. the latency, measured in time units).
+- MM_APP_ANOMALY_DETECTED &mdash; An alert based on user-defined metrics/results.
+- MM_APP_ANOMALY_SUSPECTED &mdash; An alert based on user-defined metrics/results.
+- FAILED &mdash; The job failed.
+
+
+## Creating an alert without a template
```

Comment:
```suggestion
## Creating an alert
```
as this is the basic behavior, so no need to specify that it is without a template